features/fixtures/simple_app/crystalball.yml

```diff
@@ -4,11 +4,11 @@ execution_map_path: 'execution_map.yml'
 examples_limit: 1
 #
 # Custom prediction builder class to use. Default: 'Crystalball::RSpec::StandardPredictionBuilder'
-# prediction_builder_class_name: 'Crystalball::RSpec::StandardPredictionBuilder'
+prediction_builder_class_name: 'PredictionBuilder'
 # #
 # Set of requires. Usually used to require custom predictor class file. Default: []
-# requires:
-  # - './standard_prediction_builder'
+requires:
+  - './prediction_builder'
```

Comment:
can't we override env value for your specific feature instead of customizing a config for all features?